schema.go

```diff
@@ -2046,7 +2046,116 @@ var Schema = schema.NewTypedScopeSchema[*Config](
 	// region SecretVolumeSource
 	schema.NewStructMappedObjectSchema[v1.SecretVolumeSource](
 		"SecretVolumeSource",
-		map[string]*schema.PropertySchema{},
+		map[string]*schema.PropertySchema{
+			"secretName": schema.NewPropertySchema(
+				schema.NewStringSchema(nil, nil, nil),
+				schema.NewDisplayValue(
+					schema.PointerTo("secretName"),
+					schema.PointerTo("secretName is the name of the secret in the pod's namespace to use."+
+						" More info: https://kubernetes.io/docs/concepts/storage/volumes#secret"),
+					nil,
+				),
+				false,
+				nil,
+				nil,
+				nil,
+				nil,
+				nil,
+			),
```

Comment:
Question: I explicitly declared zero values for these fields, however, I'm also happy to switch them to be implicit, not sure what the preferred code style here is.

Reply:
Never Mind, missing arugment errors